external-crates/move/crates/move-analyzer/trace-debug/src/extension.ts

```diff
@@ -215,14 +226,34 @@ async function findTraceInfo(editor: vscode.TextEditor): Promise<string> {
         throw new Error(`Cannot find package root for file  '${editor.document.uri.fsPath}'`);
     }
 
-    const pkgModules = findModules(editor.document.getText());
-    if (pkgModules.length === 0) {
-        throw new Error(`Cannot find any modules in file '${editor.document.uri.fsPath}'`);
+    let tracedFunctions: string[] = [];
+    if (path.extname(editor.document.uri.fsPath) === '.move') {
```

Comment:
Maybe use `MOVE_FILE_EXT` (and friends) here.

Reply:
This is a separate TS package so I can't re-use them but I can certainly introduce them as constants here as well!